### ISSUE TITLE
Again details about how pretrained models are trained?

### ISSUE THREAD

**Issue by YYangZiXin** [pytorch/vision#4115]
1. I use [v0.5.0/references](https://github.com/pytorch/vision/blob/build/v0.5.0/references/classification/train.py) to train a resnet50 wirth defalut config. But, I got Best_val Top1=75.806%, which has a gap of 0.3% about the pretrained model. How can I to repreduce your accuracy ?
2. I notice you said you use you recompute the batch norm statistics after training, can you show more details?

**Comment by YYangZiXin** [comment]
The lr_scheduler in your scripts decay lr at 30,60,90epoch ,but you have 90epochs totally.

**Comment by datumbox** [comment]
For information on how the models are trained and their exact params, see the [readme file](https://github.com/pytorch/vision/tree/build/v0.5.0/references/classification#resnext-50-32x4d).

Concerning the acc diff, it's worth noting that the training scripts don't fix the seed and hence every time you train your initialization is different. As expected different initialization values lead to different accuracies. The difference is usually not massive but it is within the gap you describe. 

> I notice you said you use you recompute the batch norm statistics after training, can you show more details?

I'm not sure what you mean. Could you provide references?
